src/index.tsx

```diff
@@ -25,21 +26,21 @@ const queryClient = new QueryClient({})
 
 const App = (): JSX.Element => {
   return (
-    <SafeAreaProvider>
+    <NativeBaseProvider theme={theme} colorModeManager={colorModeManager} config={nativeBaseConfig}>
       <NotificationsProvider>
-        <QueryClientProvider client={queryClient}>
-          <AuthProvider>
+        <AuthProvider>
```

Comment:
When developing in the future AuthProvider might be using react query and it will not work when it's outside provider